ynr/apps/bulk_adding/templates/bulk_add/add_form.html

```diff
@@ -13,13 +13,13 @@
 {% endblock %}
 
 {% block content %}
-<h2>Add candidates to {{ election_obj.name }}: <a href="{% url 'constituency' election=election_obj.slug post_id=post_extra.slug ignored_slug=post_extra.base.area.name|slugify %}">{{ post_extra.base.area.name }}</a></h2>
+<h2>Add candidates to {{ election_obj.name }}: <a href="{{ post_election.get_absolute_url }}">{{ post_extra.base.label }}</a></h2>
```

Comment:
This one is giving me `<a href="">` in dev

Reply:
Whoops, I managed to miss the rename of the file that tested this view. Pushed a fix so it's tested and fixed now.